package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "conforma-server",
-  "version": "0.6.1-30",
+  "version": "0.8.0",
```

Comment:
There was a migration for 0.7 but version is set at 0.6, was that an issue ?

Reply:
The version here shouldn't be changed except when we do releases/builds. The latest migrations should always be higher than the current version (but less or equal to the next intended release). The migrations are all safe in that they can be re-run over and over without causing problems (they either fail silently if they try to change something that's already changed, or we've put an explicit check before executing).

Reply:
I changed back to what's on develop: c85e0b83761e78f7e4c80dda10d7ba28ac98716c

But I don't understand, there is migration for 0.7 before i added 0.8, but the version was below that, so: ` latest migrations should always be higher than the current version (but less or equal to the next intended release)` wasn't true. 

So to test current migration work, temporary change package version, but don't commit ?

Reply:
> But I don't understand, there is migration for 0.7 before i added 0.8, but the version was below that, so: ` latest migrations should always be higher than the current version (but less or equal to the next intended release)` wasn't true.

Yeah, the current version is `0.6.1-pre-releaseX`. So any new migrations go in `0.7.0` block. After this stuff is merged, I'll make the new build 0.7.0. Then the next lot of migrations will go in new `<0.8.0` block.